docs/hub/security-secrets.md

```diff
@@ -15,9 +15,8 @@ def call_inference(prompt: str) -> str:
     return result
 ```
 
-To prevent this issue, we run an automated bot (Spaces Secrets Scanner) that scans for hard-coded secrets and opens a discussion (in case hard-coded secrets are found) about the exposed secrets & how to handle this problem.
+To prevent this issue, we run a tool called [TruffleHog](https://trufflesecurity.com/trufflehog) on each push you make. TruffleHog scans for hard-coded secrets and we will send you an email upon detection. You'll only receive an email for verified secrets. Verified secrets are secrets that have been verified by trufflehog, meaning that they can be used to authenticate to their given provider. Note that unverified secrets are not necessarily invalid, verification can also fail due to technical reasons, e.g. in that case of a network error.
```

Comment:
```suggestion
To prevent this issue, we run a tool called [TruffleHog](https://trufflesecurity.com/trufflehog) on each push you make. TruffleHog scans for hard-coded secrets, and we will send you an email upon detection. You'll only receive emails for verified secrets, which are the ones that have been confirmed to work for authentication against their respective providers. Note, however, that unverified secrets are not necessarily harmless or invalid: verification can fail due to technical reasons. In this case, verification will run again on subsequent commits to the repository.

TruffleHog can verify secrets that work across multiple services, it is not restricted to Hugging Face tokens.
```

Maybe? I'm kind of making up stuff here, feel free to ignore. Also, tokens for unsupported services can never be verified, right?

Reply:
I'll take a few things from your suggestion, thank you for the input!

> , verification will run again on subsequent commits to the repository.

This part is not correct, once a file is scanned it won't be rescanned unless a change is made to it.

> Also, tokens for unsupported services can never be verified, right?

Correct!